packages/cursorless-vscode-e2e/src/suite/scopeVisualizer/runBasicMultilineContentTest.ts

```diff
@@ -0,0 +1,84 @@
+import { openNewEditor } from "@cursorless/vscode-common";
+import * as vscode from "vscode";
+import { checkAndResetFakes } from "./checkAndResetFakes";
+import { injectFakes } from "./injectFakes";
+import { ExpectedArgs } from "./scopeVisualizerTest.types";
+
+/**
+ * Tests that the scope visualizer works with multiline content, by
+ * ensuring that the correct decorations are applied so that it looks
+ * like `./runBasicMultilineContentTest.png`.
+ */
+export async function runBasicMultilineContentTest() {
+  await openNewEditor(contents, {
+    languageId: "typescript",
+  });
+
+  const fakes = await injectFakes();
+
+  await vscode.commands.executeCommand(
+    "cursorless.showScopeVisualizer",
+    {
+      type: "namedFunction",
+    },
+    "content",
+  );
+
+  checkAndResetFakes(fakes, expectedArgs);
+}
+
+const contents = `
+function helloWorld() {
+
+}
+`;
+
+const expectedArgs: ExpectedArgs = {
```

Comment:
Admittedly, these are a bit tough to read.  Hopefully the image helps, and for the complex line decoration drawing aspect, we have unit tests

Reply:
Images definitely helps